pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -354,7 +353,7 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 	// poor new pilot and overwhelm it.
 	// TODO: instead of readiness probe, let endpoints connect and wait here for
 	// config to become stable. Will better spread the load.
-	<-s.throttle
+	<-s.initThrottle
```

Comment:
I suggest closing the connection if the channel is full/empty or whatever.. This is a way of us shedding load and taking in only 25 sidecars per second.

Reply:
Also, 25 ticks per second I understand.. wht is the 100 burst? the channel will have 100 capacity after 4 seconds, but at that time, you are literally unleashing 100 sidecars, when the Pilot couldn’t handle 25 sidecars

Reply:
@rshriram I think we should keep this PR simple and not change the rate limiter's behaviour.
Ultimately we want to throttle based on CPU, and make it adaptive. 
Sending resets when waiting may happen is problematic, even backlog during event driven pushes is not a good measure of sending GO-AWAY.

Let;s do all this in master.